### ISSUE TITLE
Latest Bungeecord version sends players to fallback server 

### ISSUE THREAD

**Issue by call911nowplz** [SpigotMC/BungeeCord#2877]
Hello,
I recently updated my bungeecord to the latest version to allow 1.16 players to join, After doing so, Me and all my players will get kicked after a small amount of time out of any sub-server back to lobby. 
This issue is not present in any stable build before 1.16 bungeecord. Any idea what could possibly be causing this?

**Comment by Janmm14** [comment]
Logs?

**Comment by call911nowplz** [comment]
Update: 
Issue was also with ProtocolLib. But please hear me out before saying this isnt a bungee issue.
The issue is when there are too many plugins using protocollib (assuming alot of packets?) on a server, Players will get kicked back to fallback server after 10-20s, But this issue doesn't happen on ANY bungeecord build before 1.16 support, And doesn't happen on any bungeecord fork. 
There are no longs, Just says in console that player got disconnected and sent back to fall back server.

**Comment by md-5** [comment]
>The issue is when there are too many plugins using protocollib (assuming alot of packets?) on a server, 

Unless you can provide evidence to the contrary that sounds 100% like a protocollib issue

**Comment by call911nowplz** [comment]
> > The issue is when there are too many plugins using protocollib (assuming alot of packets?) on a server,
> 
> Unless you can provide evidence to the contrary that sounds 100% like a protocollib issue

It was an anticheat depending on ProtocolLib causing the issue, Basically lots of packets involved and bungeecord for some reason couldn't handle that. 
I have removed the anticheat and it's working fine, I reported the issue to the developer, Even though older bungeecord versions didnt kick players, I believe its the anticheat's fault. Sorry for taking your time!
